ReactNativeClient/lib/joplin-renderer/MdToHtml/rules/embedvideo.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
```

Comment:
Normally this should be implemented in TypeScript, as mention in CONTRIBUTING.md. But let's say in JS is fine too, provided the implementation is correct.